bigbluebutton-html5/imports/ui/components/meeting-ended/meeting-ended-ts/queries.ts

```diff
@@ -0,0 +1,44 @@
+import { gql } from '@apollo/client';
+
+export interface Meeting {
+  learningDashboardAccessToken: string;
+  isBreakout: boolean;
+}
+
+export interface MeetingClientSettings {
+  askForFeedbackOnLogout: boolean;
+  allowDefaultLogoutUrl: boolean;
+  learningDashboardBase: string;
+}
+
+export interface UserCurrent {
+  role: string;
+}
+
+export interface MeetingEndDataResponse {
+    meeting: Meeting[];
+    meeting_clientSettings: MeetingClientSettings[];
+    user_current: UserCurrent[];
+}
+
+export const getMeetingEndData = gql`
```

Comment:
Can we combine the three queries into one? This could reduce the workload on the GraphQL server, especially since everyone will be sending this query at the same time when the meeting ends.

```gql
query getMeetingEndData {
    user_current {
      role
      meeting {
        learningDashboardAccessToken
        isBreakout
        logoutUrl
        clientSettings {
          askForFeedbackOnLogout: clientSettingsJson(path: "$.public.app.askForFeedbackOnLogout")
          allowDefaultLogoutUrl: clientSettingsJson(path: "$.public.app.allowDefaultLogoutUrl")
          learningDashboardBase: clientSettingsJson(path: "$.public.app.learningDashboardBase")
        }
      }
    }
  }
```